nautobot/ipam/models.py

```diff
@@ -240,6 +240,20 @@ class Prefix(PrimaryModel, StatusModel, RoleModelMixin):
         choices=choices.PrefixTypeChoices,
         default=choices.PrefixTypeChoices.TYPE_NETWORK,
     )
+    parent = models.ForeignKey(
+        "self",
+        blank=True,
+        null=True,
+        related_name="child_prefixes",  # Maybe child_prefixes? Align with "child_ips"?
```

Comment:
Why not `children` for parity with our other self-referential models?

Reply:
I'm a fan of that, but I did this because we're also going to have `child_ips` relationship from `IPAddress`. My thought is to avoid the ambiguity of which is which.